Audition/PlayheadPanel/jsx/DocumentHandler.jsx

```diff
@@ -48,7 +48,7 @@ function getDocumentInfo()
 //
 function setCTI(percent)
 {
-	if (app.activeDocument && app.activeDocument.reflect.name = 'WaveDocument')
+	if (app.activeDocument && app.activeDocument.reflect.name === 'WaveDocument')

```

Comment:
@bbb999 wanted to confirm with you this change is ok. I think it should be `===` instead of `=`? please correct me if I'm wrong